Packs/TOPdesk/Integrations/TOPdesk/TOPdesk.py

```diff
@@ -810,7 +828,8 @@ def list_operators_command(client: Client, args: Dict[str, Any]) -> CommandResul
     operators = client.get_list_with_query(list_type="operators",
                                            start=args.get('start', None),
                                            page_size=args.get('page_size', None),
-                                           query=args.get('query', None))
+                                           query=args.get('query', None),
+                                           fields=args.get('fields', None))
```

Comment:
I added the fields parameter to all get_list_with_query calls. But I noticed the TOPdesk API doesn't do anything with the fields for the operator call. Strange... because it is the same module within TOPdesk as branches and persons.

We can add it to the documentation of remove it here. I think they will implement it later.

Reply:
I removed the fields option from the function for operators.